invenio_rdm_records/resources/serializers/dublincore/schema.py

```diff
@@ -0,0 +1,189 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Dublin Core based Schema for Invenio RDM Records."""
+
+from __future__ import absolute_import, print_function
+
+from invenio_access.permissions import system_identity
+from invenio_vocabularies.proxies import current_service as vocabulary_service
+from marshmallow import Schema, fields, missing
+
+
+class DublinCore(Schema):
+    """Schema for Dublin Core in JSON."""
+
+    contributors = fields.Method('get_contributors')
+    titles = fields.Method('get_titles')
+    creators = fields.Method('get_creators')
+    identifiers = fields.Method('get_identifiers')
+    relations = fields.Method('get_relations')
+    rights = fields.Method('get_rights')
+    dates = fields.Method('get_dates')
+    subjects = fields.Method('get_subjects')
+    descriptions = fields.Method('get_descriptions')
+    publishers = fields.Method('get_publishers')
+    types = fields.Method('get_types')
+    sources = fields.Method('get_sources')
+    languages = fields.Method('get_languages')
+    coverage = fields.Method('get_locations')
+    formats = fields.Method('get_formats')
+
+    def get_titles(self, obj):
+        """Get titles."""
+        return [obj['metadata']['title']]
+
+    def get_identifiers(self, obj):
+        """Get identifiers."""
+        items = []
+        items.extend(
+            i['identifier'] for i in obj['metadata'].get('identifiers', [])
+        )
+        items.extend(p['identifier'] for p in obj.get('pids', {}).values())
+
+        return items
+
+    def get_creators(self, obj):
+        """Get creators."""
+        return [
+            c['person_or_org']['name']
```

Comment:
`name` is not a required value so it might be empty, see [here](https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/services/schemas/metadata.py#L63). You need to use `.get` instead

Reply:
for organizational creators it is a required field, see [here](https://github.com/inveniosoftware/invenio-rdm-records/blob/4fa591355cf8c6a94cec082e72ab8848b86e3ba2/invenio_rdm_records/services/schemas/metadata.py#L86)
for personal creators, the name field will be updated, see [here](https://github.com/inveniosoftware/invenio-rdm-records/blob/4fa591355cf8c6a94cec082e72ab8848b86e3ba2/invenio_rdm_records/services/schemas/metadata.py#L98)
i think there will always be a value for `name`